cdap-docs/build.sh

```diff
@@ -408,8 +408,16 @@ function build_license_dependency_pdfs() {
 }
 
 function build_standalone() {
```

Comment:
just to clarify - this now requires someone who wants to use this function to have hydrator-plugins cloned, and for it to be at the same level as cdap, right?